docs/DeveloperGuide.md

```diff
@@ -1,29 +1,134 @@
 # Developer Guide
 
+## Welcome!
+Welcome and thanks for joining us in building our program created to handle all of your currencies and money problems. With us you can use such functions as transfer, withdraw, add with different currencies to handle all your daily money problems. You also have the opportunity to add your own currencies, such as cryptocurrencies and etc. 
+
+Within this documentation you can find all the information needed to keep building it.
+
+## Setting up
+
+This section describes the development tools used in the creation of Currency Manager.
+
+### 2.1 Prerequisites
+  1. JDK 11
+  2. Intellij IDEA
+  3. Gradle 6.2
+
+### 2.2 Setting up the project
+  1. Fork this repo and clone the fork onto your computer
+  2. Open Intellij(if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first).
+  3. Set up the correct JDK version for Gradle
+      a) That can be done under `Configure` > `Project Defaults` > `Project Structure`
+  4. Click `Import Project`
+  5. Find the `build.gradle` file and select it. Click `OK`.
+  6. Click `Open as Project`
+  7. Click `OK` and accept the default settings.
+
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Save / Withdraw feature
+
+The save and withdraw feature is implemented based on the Wallet class data structure. 
+The Wallet class has several components: the username, the password, the default currency, the total balance, 
+and the deposit list containing different types of currency and their balances.
+
+Given below is the UML diagram of the Wallet class.
+
+![avatar](/property/WalletClass.jpg)
```

Comment:
While some information about association is provided, it would be nice if some more information about the structure (like multiplicity, labels, etc) was provided